File: exam_2019_july/FootballTournament.java
import java.util.Scanner;

public class FootballTournament{

	public static void main(String[] args){
	
		Scanner sc = new Scanner(System.in);
		
		String team = sc.nextLine();
		
		int matches = Integer.parseInt(sc.nextLine());
		
		if(matches == 0){
			System.out.println(team + " hasn't played any games during this season.");
			return;
		}
		
		int wins = 0;
		int losses = 0;
		int draws = 0;
		
		for(int i = 0; i < matches; i++){
			
			String result = sc.nextLine();
			
			if(result.equals("W")) wins += 3;
			else if (result.equals("D")) draws++;
			else losses++;	
		}
		
		System.out.printf("%s has won %d points during this season.%n",team, wins + draws);
		System.out.println("Total stats:");
		System.out.printf( "## W: %d%n", wins/3);
		System.out.printf( "## D: %d%n", draws);
		System.out.printf( "## L: %d%n", losses);
		System.out.printf("Win rate: %.2f", (wins/3) * 100 / (double)matches); 
		System.out.print("%");
	
	}
}

File: exam_2019_july/FootballTournament.cpp
#include <iostream>

using namespace std;

int main()
{
	string team;
	getline(cin,team);
	
	int matches;
	cin >> matches;
	
	if(matches == 0){
		cout << team << " hasn't played any games during this season." << endl;
		return 0;
	}
	
	int wins = 0;
	int losses = 0;
	int draws = 0;
	
	for(int i = 0; i < matches; i++){
		
		char result;
		cin >> result;
		
		if(result == 'W') wins += 3;
		else if(result == 'D') draws++;
		else losses++;
	}
	
	cout.setf(ios::fixed);
	cout.precision(2);
	
	cout <<  team << " has won " << wins + draws << " points during this season." << endl;
	cout << "Total stats:" << endl;
	cout << "## W: " << wins/3 << endl;
	cout << "## D: " << draws << endl;
	cout << "## L: " << losses << endl;
	cout << "Win rate: " << (wins/3) * 100 / (double)matches << "%" << endl;
	
	return 0;
}